src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -112,8 +114,9 @@ export class FindAndReplacePlugin extends UIPlugin {
 
   finalize() {
     if (this.isSearchDirty) {
-      this.refreshSearch();
+      this.refreshSearch(this.jumpToMatchSheet);
```

Comment:
I think Adrien is right, `this.jumpToMatchSheet` isn't needed :wink: 
```suggestion
      this.refreshSearch(false);
```